packages/KButton/KButton.vue

```diff
@@ -152,7 +152,7 @@ export default {
     },
 
     iconColor () {
-      if (this.$attrs.disabled !== undefined) {
+      if (this.$attrs.disabled !== undefined && this.$attrs.disabled !== false) {
```

Comment:
**[Suggestion]:** Simplified version:

```ts
if (this.$attrs.disabled) {
...
}
```

Reply:
I assumed there was a specific reason that syntax was not used previously. If there are no objections I'm happy to change it

Reply:
You could also do `if (!!this.$attrs.disabled)`